modelon/impact/client/sal/workspace.py

```diff
@@ -170,3 +170,23 @@ def experiment_create(
             **({"userData": user_data} if user_data is not None else {}),
         }
         return self._http_client.post_json(url, body=body)
+
+    def shared_definition_get(self, workspace_id: str, strict: bool = False):
+        url = (
+            self._base_uri / f"api/workspaces/{workspace_id}/"
+            f"sharing-definition?strict={'true' if strict else 'false'}"
+        ).resolve()
+        return self._http_client.get_json(url)
+
+    def import_from_shared_definition(self, shared_definition: Dict[str, Any]):
```

Comment:
I guess this method should also take selected_matchings as an optional input argument?